app/routes/_gcn.circulars/circulars.lib.ts

```diff
@@ -45,14 +59,26 @@ const subjectMatchers: SubjectMatcher[] = [
   [/ZTF[.\s_-]*(\d{2}[a-z]*)/i, ([, id]) => `ZTF${id.toLowerCase()}`],
   [/HAWC[.\s_-]*(\d{6}A)/i, ([, id]) => `HAWC-${id.toUpperCase()}`],
   [
-    /LIGO\/Virgo[.\s_-]*([S|G|GW])(\d{6}[a-z]+)/i,
+    /\.*LIGO\/Virgo[/KAGRA]?[-_ ]?([S|G|GW])(\d{5,6}[a-z]{0,2}).*/i,
     ([, flag, id]) => `LIGO/Virgo ${flag.toUpperCase()}${id.toLowerCase()}`,
   ],
```

Comment:
Why the pattern `\.*`, which will match any repeated sequence of periods?

We should not normalize `LIGO/Virgo/KAGRA 123456` to `LIGO/Virgo 123456`. LIGO, Virgo, and KAGRA are three different GW experiments that share data; removing one of them would very likely rub some people the wrong way.